server/odc-service/src/main/java/com/oceanbase/odc/config/CommonSecurityProperties.java

```diff
@@ -53,6 +53,7 @@ public class CommonSecurityProperties {
             "/api/v2/bastion/encryption/decrypt",
             "/api/v2/internal/file/downloadImportFile",
             "/api/v2/info",
+            "/api/v2/task/result",
```

Comment:
security risk

Reply:
we can limit localhost and inner ip access this api